frontend/components/draft/lesson/blocknote-editor-client.tsx

```
@@ -154,6 +229,32 @@ export function BlockNoteEditorClient({
+    view.state.doc.descendants((node, pos) => {
+      if (!node.isText) return;
+      const linkMark = node.marks.find((m) => m.type === linkMarkType);
+      if (!linkMark) return;
+      const href = (linkMark.attrs as { href?: string }).href ?? "";
+      if (isAutolinkFalsePositive(node.text ?? "", href)) {
+        tr.removeMark(pos, pos + node.nodeSize, linkMarkType);
+        changed = true;
+      }
+    });
+    if (changed) view.dispatch(tr);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)